tests/data/format/no_whitespace_stripper/module_docstring.py.out

```diff
@@ -0,0 +1,18 @@
+"""  A multi-line  .
+
+docstring  
+"""
+
+"""  A multi-line  .
+
+docstring  
+   
+"""
+
+"""  My docstring.
+
+ 
+    
+   My indented section  
+  
```

Comment:
This seems to be a very weird edge case. Shouldn't L17 have as many whitespaces as L12 has in the original file?

Reply:
I think It does ...

(screenshot from my vscode just to render the whitespace ...)
![image](https://user-images.githubusercontent.com/8116459/189731027-16511da8-3115-433c-98bf-c6130ee78f67.png)

Reply:
I believe it does (at least when looking at it in vscode to render the whitespace)

Reply:
Ah yeah, it does indeed. I should have checked the raw file myself.